src/audio_unit/pipeline.rs

```diff
@@ -19,8 +20,10 @@ impl Pipeline {
             Ok(Self { audio_units })
         }
     }
+}
 
-    pub fn process(&mut self, input: &[f32], output: &mut [f32]) -> Result<()> {
+impl AudioUnit for Pipeline {
+    fn process(&mut self, input: &[f32], output: &mut [f32]) -> Result<()> {
```

Comment:
This seems related to the comment I made on the last PR? There wasn't a need to "forward" the `.process()` method?

Reply:
Yeah, we can just use this one all the time (at the cost of having to import the `AudioUnit` trait to do it).